notebooks/image_examples/image_classification/Front Page DeepExplainer MNIST Example.ipynb

```diff
@@ -229,46 +136,22 @@
    "outputs": [],
    "source": []
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": []
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": []
-  },
   {
    "cell_type": "code",
    "execution_count": 2,
    "metadata": {},
    "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "WARNING:tensorflow:From /home/slundberg/projects/shap/shap/explainers/_deep/deep_tf.py:235: set_learning_phase (from tensorflow.python.keras.backend) is deprecated and will be removed after 2020-10-11.\n",
-      "Instructions for updating:\n",
-      "Simply pass a True/False value to the `training` argument of the `__call__` method of your layer or model.\n"
-     ]
-    },
     {
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "Using TensorFlow backend.\n",
-      "keras is no longer supported, please use tf.keras instead.\n"
+      "IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
```

Comment:
You have a couple warnings here. Can you please update juypter and ipywidgets to remove these?

Reply:
Yes, these warnings got removed after updating the jupyter and ipywidgets.